src/main/java/org/opensearch/flowframework/util/RestHandlerUtils.java

```diff
@@ -19,8 +20,16 @@
  */
 public class RestHandlerUtils {
 
+    /** Path to credential field **/
```

Comment:
Will this work even if create connector node is last node in template?

Reply:
Why wouldn't it? We are adding the filter on the SearchSourceBuilder itself, order doesn't matter. 
To be more assured tested with the below template

```json
{
    "name": "Deploy Claude Model",
    "description": "Deploy a model using a connector to Claude",
    "use_case": "PROVISION",
    "version": {
        "template": "1.0.0",
        "compatibility": ["2.12.0", "3.0.0"]
    },
    "workflows": {
        "provision": {
            "nodes": [{
                "id": "create_ingest_pipeline",
                "type": "create_ingest_pipeline",
                "user_inputs": {
                    "pipeline_id": "nlp-pipeline",
                    "configurations": {
                        "description": "test-pipeline",
                        "processors": [{
                            "text_embedding": {
                                "model_id": "jyKh544B161srMZWwY93",
                                "field_map": {
                                    "passage_text": "passage_embedding"
                                }
                            }
                        }]
                    }
                }
            }, {
                "id": "create_claude_connector",
                "type": "create_connector",
                "user_inputs": {
                    "name": "Claude Instant Runtime Connector",
                    "version": "1",
                    "protocol": "aws_sigv4",
                    "description": "The connector to BedRock service for Claude model",
                    "actions": [{
                        "headers": {
                            "x-amz-content-sha256": "required",
                            "content-type": "application/json"
                        },
                        "method": "POST",
                        "request_body": "{ \"prompt\":\"${parameters.prompt}\", \"max_tokens_to_sample\":${parameters.max_tokens_to_sample}, \"temperature\":${parameters.temperature},  \"anthropic_version\":\"${parameters.anthropic_version}\" }",
                        "action_type": "predict",
                        "url": "https://bedrock-runtime.us-west-2.amazonaws.com/model/anthropic.claude-instant-v1/invoke"
                    }],
                    "credential": {
                        "access_key": "PUT_YOUR_ACCESS_KEY_HERE",
                        "secret_key": "PUT_YOUR_SECRET_KEY_HERE"
                    },
                    "parameters": {
                        "endpoint": "bedrock-runtime.us-west-2.amazonaws.com",
                        "content_type": "application/json",
                        "auth": "Sig_V4",
                        "max_tokens_to_sample": "8000",
                        "service_name": "bedrock",
                        "temperature": "0.0001",
                        "response_filter": "$.completion",
                        "region": "us-west-2",
                        "anthropic_version": "bedrock-2023-05-31"
                    }
                }
            }]
        }
    }
}
```

Reply:
meant to type to add test cases more so, I am okay with manual test too so we can get this in